### PR TITLE
Add ability to retrieve ZFS events

### PR DESCRIPTION
**Pull request by sonicaj**
This PR exposes `zpool_events_next` from libzfs. Consumer will be able to retrieve ZFS events similar to `zpool events -fv`.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -364,6 +364,21 @@ AC_LINK_IFELSE([AC_LANG_PROGRAM(
 )
 
 
+dnl Checking function zpool_events_next existence
+
+AC_MSG_NOTICE([Checking existence of zpool_events_next existence in libzfs.h header])
```
Comment (ghost) [inline configure.ac]:
```suggestion
AC_MSG_NOTICE([Checking existence of zpool_events_next in libzfs.h header])
```

Comment (sonicaj) [reply configure.ac]: Thank you for catching that ;)

File: libzfs.pyx [hunk 2]
```diff
@@ -387,6 +387,27 @@ cdef class ZFS(object):
...
+
+        def zpool_events_single(self, zfs_dev_fd):
+            cdef nvpair.nvlist_t *nvl
+            cdef NVList py_nvl
+            cdef int zevent_fd, ret, dropped
+            zevent_fd = zfs_dev_fd
+            with nogil:
+                ret = libzfs.zpool_events_next(self.handle, &nvl, &dropped, 0, zevent_fd)
+                if ret != 0 or nvl == NULL:
+                    raise self.get_error()
```
Comment (william-gr) [inline libzfs.pyx:407]: I wasnt sure if you can call get_error and raise in a with nofil block?

**Review by william-gr — APPROVED** [review]

File: libzfs.pyx (hunk 2 above)
Comment (sonicaj) [reply libzfs.pyx:407]: Yup it works

**Pull request merged**